doc/examples/registration/plot_register_affine.py

```diff
@@ -0,0 +1,187 @@
+"""
+=================================================
+Estimate affine transformation to register images
```

Comment:
Feel free to improve! I just recommend writing a title at the imperative mood, that's descriptive of what the tutorial/demo/example will show.